src/global/Constants.h

```diff
@@ -83,7 +83,9 @@ static const std::string WARNING_ASCII_ONLY_PREFIXES =
     "You explicitly requested the ascii-prefixes-only settings or the ctre "
     "regex engine for Tokenization. This means "
```

Comment:
"You requested the CTRE parser for tokenization (either via ascii-prefixes-only = true in the settings.json or by requesting it explicitly in TurtleParserMain). This means that the input Turtle data may only use characters from the ASCII range and that no escape sequences may be used in prefixed names (e.g., rdfs:label\\,el is not allowed). This is stricter than the SPARQL standard but makes parsing faster. It works for many Turtle dumps, e.g. that from Wikidata."